.circleci/config.yml

```diff
@@ -227,6 +239,7 @@ workflows:
           requires:
             - deci-common/persist_version_info
             - deci-common/codeartifact_login
+      - integration_test
```

Comment:
Don't we want it to run only after the build? otherwise, integration of what are we testing?

Reply:
If the integration test did not pass, do we still want to release?

Reply:
@strelok899

Reply:
it is after the build .
maybe you looking on old code

Reply:
I fixed it